### PR TITLE
Prep for Ember data: misc #2

### PR DESCRIPTION
**Pull request by Monkeychip**
I have a [new ](https://github.com/hashicorp/vault/pull/24760/commits) WIP ember upgrade branch that has Ember exam merged in. While creating that branch, I was able to confirm several test changes that are still needed but are not upgrade dependent. Regardless of the ember data upgrade these changes are good housekeeping.

- [x] Enterprise test pass.

### CODE REVIEW

File: ui/lib/pki/addon/components/page/pki-overview.hbs
Comment (Monkeychip) [inline ui/lib/pki/addon/components/page/pki-overview.hbs:1]: I made an edit to this file via the github code editor and it always adds in the extra space at the bottom of the file.

File: ui/tests/acceptance/dashboard-test.js [hunk 1]
```diff
@@ -308,26 +306,27 @@ module('Acceptance | landing page dashboard', function (hooks) {
     };
 
     test('shows the correct actions and links associated with database', async function (assert) {
-      await mountSecrets.enable('database', 'database');
-      await newConnection('database');
+      const databaseBackend = `db-database`;
```
Comment (hashishaw) [inline ui/tests/acceptance/dashboard-test.js:309]: we should probably use a uuid here

**Review by hashishaw — APPROVED** [review]
😍 Just one minor comment, otherwise LGTM

File: ui/tests/acceptance/dashboard-test.js (hunk 1 above)
Comment (Monkeychip) [reply ui/tests/acceptance/dashboard-test.js:309]: Good call! I'll amend.

**Pull request merged**